<ui version="4.0" >
 <class>MultimediaDlg</class>
 <widget class="QDialog" name="MultimediaDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Dream Multimedia</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="res/icons.qrc" >
    <normaloff>:/icons/MainIcon.png</normaloff>:/icons/MainIcon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="JLBrowser" name="textBrowser" />
     </item>
     <item>
      <widget class="QLabel" name="PixmapFhGIIS" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap" >
        <pixmap>image1</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PixmapLogoJournaline" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="pixmap" >
        <pixmap>image2</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabel1" >
       <property name="text" >
        <string>Receiving Status:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CMultColorLED" native="1" name="LEDStatus" >
       <property name="minimumSize" >
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="ButtonJumpBegin" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="res/icons.qrc" >
         <normaloff>:/icons/JumpBegin.png</normaloff>:/icons/JumpBegin.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ButtonStepBack" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="res/icons.qrc" >
         <normaloff>:/icons/StepBack.png</normaloff>:/icons/StepBack.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelCurPicNum" >
       <property name="font" >
        <font>
         <family>Arial Black</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>0/0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ButtonStepForward" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="res/icons.qrc" >
         <normaloff>:/icons/StepForw.png</normaloff>:/icons/StepForw.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ButtonJumpEnd" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="res/icons.qrc" >
         <normaloff>:/icons/JumpEnd.png</normaloff>:/icons/JumpEnd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer5_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMultColorLED</class>
   <extends>QWidget</extends>
   <header>MultColorLED.h</header>
  </customwidget>
  <customwidget>
   <class>JLBrowser</class>
   <extends>QTextBrowser</extends>
   <header>jlbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/icons.qrc" />
 </resources>
 <connections/>
</ui>
